tests/test_integration_http.py

```diff
@@ -284,7 +285,7 @@ def test_inproc_crash_http(cmake, httpserver, build_args):
     child = run(
         tmp_path,
         "sentry_example",
-        ["log", "start-session", "attachment", "crash"],
+        ["log", "start-session", "attachment", "attach-view-hierarchy", "crash"],
```

Comment:
While I do understand that nothing changed in this PR wrt `crashpad` backend, I still think it makes sense to add a test for crashpad as well:

1. to automatically verify it works :-)
2. to ensure that future changes to `crashpad` attachment handling won't break it
3. to have an explicitly stated contract in form of test, that says: crashpad doesn't send `application/json` and we verified that it works in relay.

Reply:
I've added it to the Crashpad tests in https://github.com/getsentry/sentry-native/pull/1191/commits/aef02036d2ca6145f672f7d2fe003908da732df5 .